app/helpers/samples_helper.rb

```diff
@@ -288,8 +270,14 @@ def job_stats_get(pipeline_run_id)
   end
 
   def report_ready_multiget(pipeline_run_ids)
-    # query db to get ids of pipeline_runs for which "report_ready?" is true
-    PipelineRun.where(id: pipeline_run_ids).where("job_status = '#{PipelineRun::STATUS_CHECKED}' or id in (select pipeline_run_id from pipeline_run_stages where pipeline_run_stages.step_number = pipeline_runs.ready_step and pipeline_run_stages.job_status = '#{PipelineRun::STATUS_LOADED}')").pluck(:id)
+    # query db to get ids of pipeline_runs for which the report is ready
+    report_ready_result_status = "\"#{PipelineRun::REPORT_READY_OUTPUT}\":\"#{PipelineRun::STATUS_LOADED}\""
```

Comment:
Is there a comment documenting the states and state transitions of a pipeline run?    I am finding these strings difficult to read.   Not obvious why we have REPORT_READY_OUTPUT as well as STATUS_LOADED here.   Having one place that lists the states and state transitions would greatly aid in understanding.

Reply:
ok, added description in pipeline_run.rb